c++/src/kj/compat/http.c++

```diff
@@ -2305,6 +2305,7 @@ public:
       }
     });
 
+    receivedBytes += payloadLen;
```

Comment:
Added to the calls of `stream->tryRead()`. I noticed that this changes the byte counts a little bit in my test. I think it is because now we're counting the full message size including headers, not just the payload size. I'd say this behavior is probably better.